recipes/pylibjpeg-rle/recipe.yaml

```
@@ -0,0 +1,51 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  name: pylibjpeg-rle
+  python_name: pylibjpeg_rle
+  version: 2.1.0
+  python_min: "3.10"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ python_name }}-${{ version }}.tar.gz
+  sha256: 3ccba7b912f5bc0facc0f2738a5a55dd843653d2f3c02cc31a09fb5efad4b9c7
+
+build:
+  number: 0
```

```suggestion
  number: 0
  skip: py39
```
This will skip Python 3.9 for native builds.